tomls/omnibus-arbitrum-sepolia/collaterals/susde.toml

```diff
@@ -1,17 +1,9 @@
-[setting.sUSDe_max_collateral_limit]
-defaultValue = "<%= parseEther(String(0)) %>"
-
-[setting.sUSDe_issuance_ratio]
-defaultValue = "<%= parseEther('2.00') %>"
-
-[setting.sUSDe_liquidation_ratio]
-defaultValue = "<%= parseEther('1.05') %>"
-
-[setting.sUSDe_liquidation_reward]
-defaultValue = "<%= parseEther('1') %>"
-
-[setting.sUSDe_min_delegation]
-defaultValue = "<%= parseEther('100') %>"
+[var.susde_collateral_config]
+susde_max_collateral_limit = "<%= parseEther(String(3_000_000)) %>"
```

Comment:
as per sccp 356 we expect this value to be 10 mil.

Reply:
bump on this 😅  not updated

Reply:
done